main/src/cgeo/geocaching/settings/CredentialsPreference.java

```diff
@@ -135,6 +136,9 @@ protected boolean isAuthorized() {
 
     @Override
     protected void revokeAuthorization() {
+        if (credentialsMapping == CredentialActivityMapping.GEOCACHING) {
+            Cookies.clearCookies();
```

Comment:
Just wondering if the clearing has any negative effect on the other authorizations? But generally I'm completely fine with the change.